features/ouders.feature

```diff
@@ -19,236 +19,234 @@ Rule: De ouder in categorie 2 heeft ouderAanduiding "1" en de ouder in categorie
     Als personen wordt gezocht met de volgende parameters
     | naam                | waarde                                       |
     | type                | RaadpleegMetBurgerservicenummer              |
-    | burgerservicenummer | 999993653                                    |
+    | burgerservicenummer | 000007890                                    |
     | fields              | ouders.ouderAanduiding,ouders.naam.voornamen |
     Dan heeft de response een persoon met een 'ouder' met de volgende gegevens
-    | naam            | waarde |
-    | ouderAanduiding | 1      |
-    En heeft de 'ouder' de volgende 'naam' gegevens
-    | naam      | waarde  |
-    | voornamen | Lisette |
+    | naam            | waarde  |
+    | ouderAanduiding | 1       |
+    | naam.voornamen  | Lisette |
     En heeft de persoon een 'ouder' met de volgende gegevens
-    | naam            | waarde |
-    | ouderAanduiding | 2      |
-    En heeft de 'ouder' de volgende 'naam' gegevens
-    | naam      | waarde    |
-    | voornamen | Guîllaumé |
+    | naam            | waarde    |
+    | ouderAanduiding | 2         |
+    | naam.voornamen  | Guîllaumé |
 
 @gba
 Rule: de actuele gegevens van ouders worden geleverd
 
   @gba
   Scenario: oudergegevens gecorrigeerd
-    Gegeven de persoon met burgerservicenummer '999996186' heeft een ouder '1' met de volgende gegevens
+    Gegeven de persoon met burgerservicenummer '000007891' heeft een ouder '1' met de volgende gegevens
     | voornamen (02.10)  |
     | Chantal            |
     En de persoon heeft een ouder '2' met de volgende gegevens
```

Comment:
indicatie onjuist moet je hier niet opnemen. dat doen we door in de volgende stap "gecorrigeerd" te gebruiken in plaats van "gewijzigd"

Reply:
Ga ik in een nieuwe PR aanpassen.